### PR TITLE
Add all liht components

### PR DESCRIPTION
**Pull request by UnknownSilicon**
Michael sucks (he wouldn't make the pull request)

### CODE REVIEW

File: src/org/robockets/steamworks/LightsColors.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package org.robockets.steamworks;
+
+public enum LightsColors {
+	YELLOW,
+	BLUE,
+	GREEN,
+	PURPLE,
+	WHITE,
+	BROWN;
+
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/LightsColors.java]: Remove new line

File: src/org/robockets/steamworks/Robot.java [hunk 2]
```diff
@@ -41,14 +43,18 @@
...
 	public static Command autonomousCommand;
 	public static Command autoTest;
 	public static Command drive;
 	public static Command climb;
 	public static Command toggleDriveMode;
+	public static Command cylonCommand = new Cylon();
+
 	private SendableChooser<Command> autonomousChooser;
 	
+
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/Robot.java]: Only one new line here

File: src/org/robockets/steamworks/commands/ButtonPress.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
+package org.robockets.steamworks.commands;
+
+
+
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/commands/ButtonPress.java]: Remove 2 new lines

File: src/org/robockets/steamworks/subsystems/LED.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package org.robockets.steamworks.subsystems;
+
+
+
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/subsystems/LED.java]: Remove 2 new lines

**Review by UnknownSilicon — COMMENTED** [review]
@michaelcapone21 Add some javadocs

**Review by UnknownSilicon — COMMENTED** [review]
:+1:

**Review by bshin100 — APPROVED** [review]
Other than removing those print statements, 👍

**Pull request merged**